consul/backup.sls

```diff
@@ -8,20 +8,20 @@
 {%- set s3_bucket   = salt['pillar.get']('consul_backup:s3:bucket') %}
 {%- set s3_path     = salt['pillar.get']('consul_backup:s3:path', 'consul') %}
 {%- set s3_profile  = salt['pillar.get']('consul_backup:s3:profile', 'default') %}
-{%- set user        = salt['pillar.get']('consul_backup:user', 'consul') %}
+{%- set user        = salt['pillar.get']('consul_backup:user', 'root') %}
 
 {%- set default_exec_path = '/usr/local/bin/backup-consul.sh' %}
 {%- set exec_path = salt['pillar.get']('consul_backup:exec_path', default_exec_path) %}
 
-{%- set default_local_path = '/home/consul/backup' %}
+{%- set default_local_path = '/etc/consul/backup' %}
```

Comment:
backups are not config, we should find another place for this, `/etc/` is not the right place.

Reply:
Yes but the backups are also part of the consul then to make it simple to find when you have similar files in the same folder.  But probably good place `/backups/consul` is a good place and easy to find them

Reply:
we can keep them in the same folder in `$HOME/consul/backups` make more sense.